src/components/camera/camera.tsx

```diff
@@ -58,7 +56,7 @@ export class CameraPWA {
   flashMode: FlashMode = 'off';
 
   async componentDidLoad() {
-    if (this.isServer) {
+    if (Build.isServer) {
```

Comment:
This `context` API option was removed from Stencil in favor of `!Build.isBrowser` [(source)](https://github.com/ionic-team/stencil/blob/main/BREAKING_CHANGES.md#isserver). 

Later, a new build conditional called `Build.isServer` was introduced: https://github.com/ionic-team/stencil/commit/56d94f33db5917e402692cf1740e0b62047e4e63